tests/Infrastructure/Repositories/LocalFilesRepositoryTest.php

```diff
@@ -79,6 +79,12 @@ public function testPassFileInsteadOfDirectory(): void
 
         self::assertCount(1, $files);
         self::assertInstanceOf(SplFileInfo::class, $files[0]);
-        self::assertStringContainsString('/Fixtures/FileToInspect.php', $files[0]->getRealPath());
+        $path = $files[0]->getRealPath();
+
+        if ($path === false) {
+            self::fail("Path cannot be false.");
```

Comment:
Didn't know about this one :+1:

Reply:
Apparently phpstan wouldn't let it pass without checking this. Weird that it haven't failed before now though. 😕

Reply:
A new phpstan version has been released this night. That might explain this :wink:

Reply:
Ah nice catch, must be it. 😆